package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.100.0",
+  "version": "2.102.0",
```

Comment:
You skipped a version 🔢  😛

Reply:
Actually, it looks like `2.101` was just published, so this is the right version.

Reply:
I skipped one because of a separate PR I was planning on merging first, but then John beat me to it so now I have to bump both PRs :shakesfist: